_collections/_information/helpful-links.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default-md
+title: Helpful Links
+subtitle: Use the resources below to learn more about the the Boy Scouts of America and the Atlanta Area Council and access useful training documents and videos. Access additional information and forms on our <a href="/information/forms-documents.html">Forms &amp; Documents page</a>.
+nav_order: 7
+redirect_from: helpful-links.html
+column-count: 2
+---
+## Boy Scouts of America
+
+Visit the central website for the Boy Scouts of America to learn about all of the BSA scouting programs and access useful information and training.
+
+[Visit scouting.org](https://www.scouting.org/){:.mdc-button.md}
+
+## Atlanta Area Council
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/app/Troop_370/troop370/pullRequest?prid=4642321)